infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/config/LogReplicationFullTableConfig.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.infrastructure.logreplication.config;
+
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+import org.corfudb.infrastructure.ServerContext;
+import org.corfudb.runtime.LogReplication.LogReplicationSession;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+
+
+/**
+ * This class represents any Log Replication Configuration,
```

Comment:
Can we update the file level comment incorporating the Full table config? Little details about config v/s full table config would be better in readability.